src/test/java/de/caritas/cob/userservice/api/service/emailsupplier/NewMessageEmailSupplierTest.java

```diff
@@ -246,6 +253,31 @@ public void generateEmails_Should_ReturnExpectedEmailToAsker_When_ConsultantWrit
     assertThat(templateData.get(2).getValue(), is("app baseurl"));
   }
 
+  @Test
+  public void generateEmails_Should_ReturnExpectedEmailToAsker_When_ConsultantWritesToValidReceiverMultiTenancy() {
+    var tenantData = new TenantData();
```

Comment:
could you pls split these blocks into methods initializing sections, for readability purposes. 
For instance first block could be extracted to a method called givenCurrentTenantDataIsSet()

Reply:
ok